client/time_helper.c

```diff
@@ -0,0 +1,35 @@
+#include "time_helper.h"
+
+struct timeval timeval_subtract(struct timeval x, struct timeval y) {
```

Comment:
This function is marked as `static`, but actually used in a separate compilation unit. It compiles only because we are using SCU.